Assets/Scripts/GridGen.cs

```diff
@@ -174,6 +185,32 @@ public class GridGen : MonoBehaviour
         PutSprites();
     }
 
+    public void AddState() {
+        List<List<char>> tempList = new List<List<char>>();
```

Comment:
I think `currentState` would be a better name.
```suggestion
        List<List<char>> currentState = new List<List<char>>();
```